### PR TITLE
fix: country state on billing step not getting persisted upon selection

### PR DESCRIPTION
**Pull request by nentor**
## Description
Fixes a bug where the state for the country select field on the checkout billing step does not change upon selection

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: packages/theme/pages/Checkout/Billing.vue [hunk 1]
```diff
@@ -121,7 +121,7 @@
         >
           <SfSelect
             v-e2e="'billing-country'"
-            :value="billingDetails.country"
+            v-model="billingDetails.country"
```
Comment (Baroshem) [inline packages/theme/pages/Checkout/Billing.vue:124]: Could you please create an appriopriate changelog entry similar to other features we had? :)

**Review by Baroshem — COMMENTED** [review]
One request from me :)

File: packages/theme/pages/Checkout/Billing.vue (hunk 1 above)
Comment (nentor) [reply packages/theme/pages/Checkout/Billing.vue:124]: Done

**Review by Baroshem — APPROVED** [review]

**Comment by Baroshem** [comment]
Thanks @nentor !

**Pull request merged**